pages/submission/PKPSubmissionHandler.inc.php

```diff
@@ -89,6 +89,8 @@ function wizard($args, $request) {
 		$step = isset($args[0]) ? (int) $args[0] : 1;
 		$templateMgr->assign('step', $step);
 
+		$templateMgr->assign('sectionId', $request->getUserVar('sectionId')); // to add a sectionId parameter to tab links in template
```

Comment:
I don't think it's a particular risk here, but best to force an `(int)` cast when reading in numeric IDs from the user. (Blanks/nonexistent parameters get cast to `0`, which evaluate to boolean `false`, so they are generally OK too.)

Reply:
Makes sense, will do.